Resources/Private/Language/de.locallang_db.xlf

```diff
@@ -405,6 +405,10 @@
 			<source>checkbox</source>
 			<target>Checkbox</target>
 		</trans-unit>
+		<trans-unit id="tx_dpf_domain_model_metadataobject.input_type.hidden" approved="yes">
+			<source>hidden</source>
+			<target>Hidden</target>
```

Comment:
Translation should be "Versteckt" not "Hidden".

Reply:
The translation has been adjusted accordingly.

Reply:
The use case is exactly as described above.